src/js/controllers/backup.js

```diff
@@ -40,11 +56,8 @@ angular.module('copayApp.controllers').controller('backupController',
       return false;
     };
 
-    $scope.backTo = function(state) {
-      if (state == 'walletHome')
-        go.walletHome();
-      else
-        go.preferences();
+    $scope.backTo = function() {
```

Comment:
The function should not be called `backTo...`  and receive not params.

Maybe just `goBack()`